docs/examples/uds_auxiliary.rst

```diff
@@ -0,0 +1,295 @@
+    N_TA=0xFF
+    N_AE=0xFF
+    Mtype=DIAGNOSTICS
+    discardNegResp=False
+
+    [virtual]
+    interfaceName=virtualInterface
+
+    [peak]
+    device=PCAN_USBBUS1
+    f_clock_mhz=80
+    nom_brp=2
+    nom_tseg1=63
+    nom_tseg2=16
+    nom_sjw=16
+    data_brp=4
+    data_tseg1=7
+    data_tseg2=2
+    data_sjw=2
+
+    [vector]
+    channel=1
+    appName=MyApp
+
+    [socketcan]
+    channel=can0
+
+
+Send UDS Raw Request
+--------------------
+| Send UDS request as list of raw bytes.
+| The method send_uds_raw(:py:meth:`pykiso.lib.auxiliaries.udsaux.UdsAuxiliary.send_uds_raw`) takes one mandatory parameter msg_to_send and one optional : timeout_in_s
+| The parameter msg_to_send is simply the UDS request payload which is a list of bytes.
+| The optional parameter timeout_in_s (by default fixed to 5 seconds) simply represent the maximum
+  amount of time in second to wait for a response from the device under test. If this timeout is reached, the
+  uds-auxiliary stop to acquire and log an error.
+
+The method send_uds_raw method returns a :py:class:`~ebplugins.udsaux.uds_response.UdsResponse` object, which is a subclass of `UserList
+<https://docs.python.org/3/library/collections.html#collections.UserList>`.
+UserList allow to keep property of the list, meanwhile attributes can be set, for UdsResponse, defined attributes
+refer to the positivity of the response, and its NRC if negative.
+
+.. code:: python
+
+    class UdsResponse(UserList):
+        NEGATIVE_RESPONSE_SID = 0x7F
+
+        def __init__(self, response_data) -> None:
+            super().__init__(response_data)
+            self.is_negative = False
+            self.nrc = None
+            if self.data and self.data[0] == self.NEGATIVE_RESPONSE_SID:
+                self.is_negative = True
+                self.nrc = NegativeResponseCode(self.data[2])
+
+Here is an example:
+
+
+.. code:: python
+
+    import pykiso
+    from pykiso.auxiliaries import uds_aux
+    from collections import UserList
+
+    @pykiso.define_test_parameters(suite_id=1, case_id=1, aux_list=[uds_aux])
+    class ExampleUdsTest(pykiso.BasicTest):
+        def setUp(self):
+            """Hook method from unittest in order to execute code before test case run.
+            """
+            pass
+
+        def test_run(self):
+            # Set extended session
+            diag_session_response = uds_aux.send_uds_raw([0x10, 0x03])
+            self.assertEqual(diag_session_response[:2], [0x50, 0x03])
+            self.assertEqual(type(diag_session_response), UserList)
+            self.assertFalse(diag_session_response.is_negative)
+
+        def tearDown(self):
+            """Hook method from unittest in order to execute code after test case run.
+            """
+            pass
+
+Send UDS Config Request
+-----------------------
+| Send UDS request as a configurable data dictionary. This method can be more practical for UDS requests with long payloads and a multitude of parameters.
```

Comment:
If I remembered well, this method can only be used if the command is defined in the .odx file right. Can we mention it?